src/GeositeFramework/js/Plugin.js

```diff
@@ -470,11 +470,6 @@ require(['use!Geosite',
 
             setResizable(view, pluginObject.resizable);
 
-            new ConstrainedMoveable($uiContainer[0], {
-                handle: $uiContainer.find('.sidebar-nav')[0],
-                within: true
-            });
-
```

Comment:
Consider removing `ConstrainedMoveable` from the define statement as well, if it's not used anywhere else in this file.